website/src/components/ProductHuntCard.tsx

```diff
@@ -21,8 +22,8 @@ export default function ProductHuntCard({
       to="https://www.producthunt.com/posts/docusaurus-2-0?utm_source=badge-featured&utm_medium=badge&utm_souce=badge-docusaurus-2-0"
       className={className}
       style={{display: 'block', width: 250, height: 54, ...style}}>
-      <img
-        src="https://api.producthunt.com/widgets/embed-image/v1/featured.svg?post_id=353916&theme=light"
+      <Image
+        img="https://api.producthunt.com/widgets/embed-image/v1/featured.svg?post_id=353916&theme=light"
```

Comment:
Ideal image is not designed to work with external URLs. `<img>` is fine in this case